libpod/runtime.go

```diff
@@ -714,3 +715,15 @@ func (r *Runtime) generateName() (string, error) {
 func (r *Runtime) ImageRuntime() *image.Runtime {
 	return r.imageRuntime
 }
+
+// RemoveMountOptions removes mount ouptions from the graphdriver
+func (r *Runtime) RemoveMountOptions() {
+	var opts []string
+	for _, i := range r.config.StorageConfig.GraphDriverOptions {
+		if strings.Contains(i, "mountopt") {
+			continue
+		}
+		opts = append(opts, i)
+	}
+	r.config.StorageConfig.GraphDriverOptions = opts
```

Comment:
Can we do this without modifying the storage.Store?

Reply:
Not sure how?

Reply:
It no longer modifies the store.  Just sends the options in on the container creation.